src/main/java/de/dentrassi/rpm/builder/RpmMojo.java

```diff
@@ -601,6 +602,27 @@ public void setGenerateDefaultSourcePackage(final boolean generateDefaultSourceP
     @Parameter(property = "rpm.signature")
     Signature signature;
 
+    /**
+     * An optional string specifying the compression method to use for compressing the payload of the final RPM.
+     * <p>
+     * The method name must be one of the names returned by {@link PayloadCoding#values()}.
+     * <p>
+     * If no payload coding is specified, then the default RPM compression method is used.
+     */
+    @Parameter(property = "rpm.payloadCoding")
+    String payloadCoding;
```

Comment:
Doesn't this also need the markdown documents changed?

Reply:
Added.

Reply:
If payloadCoding is `PayloadCoding.ZSTD` by default why not define it here which might make the code slightly simpler? Also the javadoc says `the default RPM compression method is used` but that isn't quite right is it?

Reply:
So the default for RPM is actually `"gzip 9"`. In Fedora, and RPM 6, they have been using `"zstd 19`". So, I am not sure what the default for RPM Builder should be.

You can check a particular RPM with

```bash
 rpm -q --queryformat '%{PAYLOADCOMPRESSOR} %{PAYLOADFLAGS}\n' foo.rpm
```